airbyte-integrations/connectors/source-mssql/mssql-script.sql

```diff
@@ -0,0 +1,256 @@
+	set @fullloop = @fullloop + 1;
+end;
+
+while @vmax > @vmaxoneinsert begin
+	set @fullloop = @fullloop + 1;
+	set @vmax = @vmax - @vmaxoneinsert;
+	set @lastinsert = @vmax;
+end;
+
+DECLARE @insertTable NVARCHAR(MAX)
+set @insertTable = CONVERT(NVARCHAR(max), 'insert into test (varchar1, varchar2, varchar3, varchar4, varchar5, longblobfield, timestampfield) values (');
+while @counter < @vmaxoneinsert begin
+	set @insertTable = CONVERT(NVARCHAR(max), concat(@insertTable, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP), ('));
+	set @counter = @counter + 1;
+end;
+set @insertTable = CONVERT(NVARCHAR(max), concat(@insertTable, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP);'));
+
+while @vmax < 1 begin
+	set @fullloop = 0
+	set @vmax = 1
+end;
+
+while @fullloopcounter < @fullloop begin
+	EXEC (@insertTable);
+	set @fullloopcounter = @fullloopcounter + 1;
+end;
+
+DECLARE @insertTableLasted NVARCHAR(max);
+set @insertTableLasted = CONVERT(NVARCHAR(max), 'insert into test (varchar1, varchar2, varchar3, varchar4, varchar5, longblobfield, timestampfield) values (');
+while @lastinsertcounter < @lastinsert begin
+	set @insertTableLasted = CONVERT(NVARCHAR(max), concat(@insertTableLasted, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP), ('));
+	set @lastinsertcounter = @lastinsertcounter + 1;
+end;
+
+set @insertTableLasted = CONVERT(NVARCHAR(max), concat(@insertTableLasted, @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @dummyIpsum, ', ', @fieldText, ', CURRENT_TIMESTAMP);'));
+
+while @lastinsert > 0 begin
+	EXEC (@insertTableLasted);
+	set @lastinsert = 0;
+end;
+
+end;
+go --
+
+create procedure table_create(@val int)
+as
+begin
+set nocount on;
+
+-- SQLINES LICENSE FOR EVALUATION USE ONLY
+create table test
+(
+id int check (id > 0) not null identity primary key,
+varchar1 varchar(255),
+varchar2 varchar(255),
+varchar3 varchar(255),
+varchar4 varchar(255),
+varchar5 varchar(255),
+longblobfield nvarchar(max),
+timestampfield datetime2(0)
+);
+
+DECLARE @extraSmallText NVARCHAR(max);
+DECLARE @smallText NVARCHAR(max);
+DECLARE @regularText NVARCHAR(max);
+DECLARE @largeText NVARCHAR(max);
+set @extraSmallText = '''test weight 50b - 50b text, 50b text, 50b text'''
+set @smallText = '''test weight 500b - 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text,
+500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text, 500b text'''
```

Comment:
Can you use the [REPLICATE](https://docs.microsoft.com/en-us/sql/t-sql/functions/replicate-transact-sql?view=sql-server-ver15) function to construct these long string? Especially for `@regularText` and `@largeText`.

Reply:
updated it